easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,57 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeNinja'
+
+name = 'flatbuffers'
+version = '2.0.7'
+
+homepage = 'https://github.com/google/flatbuffers/'
+description = """FlatBuffers: Memory Efficient Serialization Library"""
```

Comment:
Maybe description should be updated to mention that this easyconfigs includes the Python bindings as well?

Reply:
Done